packages/core/stories/accordion/accordion.stories.tsx

```diff
@@ -186,3 +188,38 @@ export const Status: StoryFn<AccordionGroupProps> = (props) => (
     ))}
   </AccordionGroup>
 );
+
+export const AdditionalLabels: StoryFn<AccordionProps> = (props) => (
+  <Accordion style={{ width: "90%" }} {...props}>
+    <AccordionHeader>
+      <StackLayout direction={"row"} style={{ width: "100%" }}>
+        <StackLayout gap={0.5} align="start">
+          <Text>Title</Text>
+          <Text color="secondary">Description goes here</Text>
+        </StackLayout>
+        <div style={{ marginLeft: "auto" }}>
+          <Text color="secondary" style={{ width: "max-content" }}>
```

Comment:
Not sure why `width` is needed

Reply:
Had added it for the use case that STORY has, that secondary label should not be wrapped if both description & secondary label both are long. 
But i will remove it from here. Channel teams can handle it on their side i guess